app/routers/dayview.py

```diff
@@ -0,0 +1,116 @@
+from datetime import datetime, timedelta
+from typing import Tuple, Union
+
+from fastapi import APIRouter, Depends, Request
+from fastapi.templating import Jinja2Templates
+from sqlalchemy import and_, or_
+
+from app.database.database import get_db
+from app.database.models import Event, User
+from app.dependencies import TEMPLATES_PATH
+
+
+templates = Jinja2Templates(directory=TEMPLATES_PATH)
+
+
+router = APIRouter()
+
+
+class DivAttributes:
+    GRID_BAR_QUARTER = 1
+    FULL_GRID_BAR = 4
+    MIN_MINUTES = 0
+    MAX_MINUTES = 15
+    BASE_GRID_BAR = 5
+    FIRST_GRID_BAR = 1
+    LAST_GRID_BAR = 101
+    DEFAULT_COLOR = 'grey'
+    DEFAULT_TIME_FORMAT = "%H:%M"
+    MULTIDAT_TIME_FORMAT = "%d/%m %H:%M"
+
+    def __init__(self, event: Event,
+                 day: Union[bool, datetime] = False) -> None:
+        self.start_time = event.start
+        self.end_time = event.end
+        self.day = day
+        self.start_multiday, self.end_multiday = self._check_multiday_event()
+        self.color = self._check_color(event.color)
+        self.total_time = self._set_total_time()
+        self.grid_position = self._set_grid_position()
+
+    def _check_color(self, color: str) -> str:
+        if color is None:
+            return self.DEFAULT_COLOR
+        return color
+
+    def _minutes_position(self, minutes: int) -> int:
+        min_minutes = self.MIN_MINUTES
```

Comment:
Is this function suppose to return None on minutes = 0 and 4 on minutes = 60?

If no, please add a test to assure this error wouldn't happen on refactor

Reply:
None on minutes = 0 and 4 on minuts = 60
בגדול בגלל שהארגומנט הוא attr של Datetime אז זה אף פעם לא יהיה 60.
בכל מקרה הוספתי טסט בעבור זה.